File: src/main/java/pl/dmichalski/algorithms/_8_naive_string_search/Runner.java
package pl.dmichalski.algorithms._8_naive_string_search;

/**
 * Write a function called countNumberOfAppearances, that accepts
 * two strings: text and searchingText and returns number of appearances
 * searchingText in the text.
 */
public class Runner {

    public static void main(String[] args) {
        String text = "Ala ma kota a nawet kota kota";
        String searchingText = "kota";

        StringSearchService searchService = new StringSearchService();
        int result = searchService.countNumberOfAppearances(text, searchingText);
        System.out.println("result = " + result);
    }

}

File: src/main/kotlin/pl/dmichalski/algorithms/algorithms/_08_naive_string_search/Runner.kt
package pl.dmichalski.algorithms.algorithms._08_naive_string_search

/**
 * Write a function called countNumberOfAppearances, that accepts
 * two strings: text and searchingText and returns number of appearances
 * searchingText in the text.
 */
object Runner {

    @JvmStatic
    fun main(args: Array<String>) {
        val text = "Ala ma kota a nawet kota kota"
        val searchingText = "kota"
        val searchService = StringSearchService()

        val result = searchService.countNumberOfAppearances(text, searchingText)

        println("Text '$text' contains text '$searchingText' $result times.")
    }

}
